app/data/events.json

```diff
@@ -1,4 +1,26 @@
 [
+	{
+		"date": "2025-04-17T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-???",
```

Comment:
@SeanSanker keeping an eye on this, but feel free to update the PR or post the URL here once we have a date and Eventbrite 🙏